freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/test/ClientTestV1Endpoint.java

```diff
@@ -17,9 +22,29 @@
 @Consumes(MediaType.APPLICATION_JSON)
 @Api(value = "/v1/freeipa/test", description = "Client test endpoint", protocols = "http,https", consumes = MediaType.APPLICATION_JSON)
 public interface ClientTestV1Endpoint {
+
     @GET
     @Path("{id}/{name}")
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = "Retrieves user information", produces = MediaType.APPLICATION_JSON, nickname = "userShowV1")
     String userShow(@PathParam("id") Long id, @PathParam("name") String name);
+
+    @POST
+    @Path("check_users")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "Checks if users exists in FreeIPA of the environment", produces = MediaType.APPLICATION_JSON, nickname = "checkUsers")
+    Boolean checkUsers(@Valid CheckUsersRequest checkUsersRequest);
```

Comment:
are you sure a boolean will be sufficient here later? I would return a response object which could be enhanced later if needed

Reply:
makes sense, i wasn't sure if i need that or not, but maybe it is worth to use response object